metals/src/main/scala/scala/meta/internal/metals/UserConfiguration.scala

```diff
@@ -146,6 +147,15 @@ object UserConfiguration {
            |
            |""".stripMargin
       ),
+      UserConfigurationOption(
```

Comment:
I'm curious if we can avoid this user configuration option by refining how we handle "import build". I would be OK with disabling the output panel popup for "import build" and instead attach a command to the status bar item to reveal the output panel.

Reply:
That's an awesome idea! We could also make sure that the option is available also if the build import failed. Something like "go to logs".

Reply:
Yea, I also agree, I think if we were able to avoid the configuration option by providing an action item to go to the logs, that would be great.